oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -310,6 +316,17 @@ public APIGatewayProxyResponseEvent handleRequest(
                             accountInterventionService.getAccountStatus(
                                     userInfo.getSubject().getValue());
                     reproveIdentity = accountStatus.reproveIdentity();
+
+                    if (accountStatus.blocked()
```

Comment:
The feature flag checks here will conflict with https://github.com/govuk-one-login/authentication-api/pull/3698

Reply:
Should be straightforward to resolve, happy to do so once https://github.com/govuk-one-login/authentication-api/pull/3698 is merged